src/main/scala/loamstream/db/slick/Tables.scala

```diff
@@ -116,16 +116,15 @@ final class Tables(val driver: JdbcProfile) extends DbHelpers with Loggable {
     //NB: Specify the length of this column so that we hopefully don't get a too-small VARCHAR,
     //and instead some DB-specific column type appropriate for strings thousands of chars long.
     def cmd = column[String]("CMD", O.Length(maxStringColumnLength))
-    def stdoutPath = column[String]("STDOUT_PATH", O.Length(maxStringColumnLength))
-    def stderrPath = column[String]("STDERR_PATH", O.Length(maxStringColumnLength))
+    def jobDir = column[Option[String]]("JOB_DIR", O.Length(maxStringColumnLength))
```

Comment:
As this likely points back to the CSV header, if you did go with the `Product` idea for writing the columns, this would likely need to be updated as well.

Reply:
This can't be done reflectively, or really, dynamically, since this part effectively sets out a DB schema via a DSL.  According to the Slick docs, these things have to be `def`s (I don't know why), and in any case, while you could infer column names from case-class field names, you'd still need some kind of mapping between all the fields and the metadata needed to build the DB schema, like what is passed to `column` here. :\